packages/wdio-static-server-service/src/launcher.ts

```diff
@@ -1,43 +1,52 @@
-import path from 'path'
-import { promisify } from 'util'
+import { join, resolve } from 'path'
+
+declare module 'util' {
+    function promisify<T1>(fn: (arg1: T1, callback: (err?: any) => void) => void): (arg1: T1) => Promise<void>;
+}
```

Comment:
Why is this needed?

Reply:
There seems to be some issue with promisify overloading , similar issue is mentioned here :
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/20497

By using the above code, we are specifying which method to overload instead of the generic one

Reply:
let's remove declare module util